graphs/omen-thumbnailmapping/src/schema.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  Entity,
+  Value,
+  ValueKind,
+  store,
+  Bytes,
+} from "@graphprotocol/graph-ts";
+
+export class OmenThumbnailMapping extends Entity {
+  constructor(marketAddress: Bytes) {
+    super();
+    this.set("marketAddress", Value.fromBytes(marketAddress));
+  }
+
+  save(): void {
+    let marketAddress = this.get("marketAddress");
+    let image_hash = this.get("image_hash");
+    assert(marketAddress != null, "Cannot save OmenThumbnailMapping entity without an marketAddress");
+    assert(image_hash != null, "Cannot save OmenThumbnailMapping entity without an image_hash");
```

Comment:
Why are these assertions necessary?
I would have expected that both vars are never empty - or is this not enforced in the smart contract?

Reply:
I think it's because this schema is used in [handleImageUpdated function here](https://github.com/gnosis/labs-contracts/blob/f6f9d1e3d91c11316d37a2c7611540b2b5dfc85d/graphs/omen-thumbnailmapping/src/contract.ts#L4), so to enforce that the schema class is called correctly.